CHANGELOG.md

```diff
@@ -14,7 +14,27 @@
 * one instance of mux in kafka plugin 
 * new field `group-id` can be added to kafka.conf. This value is used as a Group ID in order to set it 
   manually. In case the value is not provided, the service label is used instead (just like before). 
-      
+
+## Agent, Flavors
+Input arguments of `core.NewAgent()` has been changed (for backward compatibility the is `core.NewAgentDeprecated()`):
+  * it can be called without any options `core.NewAgent(flavor)`
+  * you can pass options it can be called without any options `core.NewAgent(flavor, core.WithTimeout(1* time.Second))`
```

Comment:
you can pass options to the ... or it can be called without any options - this one is a bit unclear